padguidedb/padguidedb.py

```diff
@@ -17,6 +17,7 @@
 
 PADGUIDEDB_COG = None
 
+sys.executable = "/usr/bin/python3"
```

Comment:
This seems like a bad idea. Don't override things like this in libraries your don't own. Just make it a local variable instead.